contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -89,11 +90,9 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
     /// @dev next payment due time per borrower for each pool
     mapping(ITrueFiPool2 => mapping(address => uint256)) public nextInterestRepayTime;
 
-    /// @dev whitelist for allowing pools to have lines of credit
-    mapping(ITrueFiPool2 => bool) public isPoolAllowed;
+    mapping(address => bool) private isPoolAllowed_DEPRECATED;
```

Comment:
Previously we used `DEPRECATED__` as a prefix, so let's stick to the naming that is already in use